libexec/mcd/mcd-flaps

```diff
@@ -33,16 +33,16 @@ case $1 in
       done
   };;
   beg) {
-    mcd --to-wad "$(seth call "${MCD_FLAP?}" 'beg()(uint)')"
+    mcd --to-wad "$(seth call "${MCD_FLAP?}" 'beg()')"
   };;
   ttl) {
-    seth --to-dec "$(seth call "${MCD_FLAP?}" 'ttl()(uint48)')"
+    seth --to-dec "$(seth call "${MCD_FLAP?}" 'ttl()')"
```

Comment:
`seth call "${MCD_FLAP?}" 'ttl()(uint48)'`

Reply:
Cleaned up all of flips/flaps/flops